README.md

```diff
@@ -33,4 +41,15 @@ export PATH=$PATH:$BipedalLocomotionControllers_INSTALL_DIR/lib
 # :running: How to use the libraries
 bipedal-locomotion-controllers provides native CMake support which allows the library to be easily used in CMake projects.
 
-bipedal-locomotion-controllers exports the `CMake` targets presented in [Exported components](#orange_book-exported-components) section. The targets can be imported using the `find_package` CMake command and used by calling `target_link_libraries`.
+bipedal-locomotion-controllers exports the `CMake` targets presented in
+[Exported components](#orange_book-exported-components) section. The targets can
+be imported using the `find_package` CMake command and used by calling
+`target_link_libraries`.
+For instance, the `OptimalControlUtilities` component can be used as follows:
+```cmake
+cmake_minimum_required (VERSION 3.0)
```

Comment:
Nitpick and  not a problem if you want to otherwise, but in the CMake codebase in our typical gudelines (see https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md#cmake and https://github.com/cheshirekow/cmake_format) we do not have a space between the CMake command and the brackets.

Reply:
Done